tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+) -> None:
+    _break_print("conda-forge updates ...")
+
+    if not is_release_candidate:
+        message = (
+            "NOTE: after several hours conda-forge automation will create a "
+            "Pull Request against conda-forge/iris-feedstock (via the "
+            "regro-cf-autotick-bot). Quicker to sort it now, manually ..."
+        )
+        _break_print(message)
+
+    message = (
+        "Make sure you have a GitHub fork of:\n"
+        "https://github.com/conda-forge/iris-feedstock"
+    )
+    _wait_for_done(message)
+
+    message = (
+        "Make sure you have a local clone of your iris-feedstock fork.\n"
+        "`cd` into your clone."
+    )
+    _wait_for_done(message)
+
+    if is_release_candidate:
+        upstream_branch = "release-candidate"
+    else:
+        upstream_branch = "main"
+
+    # TODO: automate
+    message = (
+        "Checkout a new branch for the conda-forge changes for this release:\n"
+        "git fetch upstream;\n"
+        f"git checkout upstream/{upstream_branch} -b {release_strings.tag};\n"
+    )
+    _wait_for_done(message)
+
+    message = (
+        "Update ./recipe/meta.yaml:\n\n"
+        f"- The version at the very top of the file: {release_strings.release}\n"
+        f"- The sha256 hash: {sha256}\n"
+        "- Requirements: align the packages and pins with those in the Iris repo\n"
+        "- Maintainers: update with any changes to the dev team\n"
+        "- MAKE SURE everything else is correct - plenty of other things "
+        "might need one-off changes.\n"
+    )
+    _wait_for_done(message)
+
+    # TODO: automate
+    message = (
+        "No other file normally needs changing in iris-feedstock, so push up "
+        "the changes to prepare for a Pull Request:\n"
+        f'git commit -am "Recipe updates for {release_strings.tag} .";\n'
+        f"git push -u origin {release_strings.tag};"
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Follow the Pull Request process to get {release_strings.tag} branch "
+        f"merged into upstream/{upstream_branch} .\n"
+        "Specific conda-forge guidance will be automatically given once the "
+        "PR is created."
+    )
+    _wait_for_done(message)
+
+    message = (
+        f"Confirm that {release_strings.release} appears in this list:\n"
+        "https://anaconda.org/conda-forge/iris/files"
+    )
+    _wait_for_done(message)
+
+    if not is_release_candidate:
+        message = (
+            f"Confirm that {release_strings.release} is displayed on this "
+            "page as the latest available:\n"
+            "https://anaconda.org/conda-forge/iris"
+        )
+        _wait_for_done(message)
+
+    if is_release_candidate:
+        channel_command = " -c conda-forge/label/rc_iris "
+    else:
+        channel_command = " "
+    message = (
+        "Confirm that conda install works as expected:\n"
+        f"conda install{channel_command}iris={release_strings.release};"
```

Comment:
Perhaps add a line suggesting mamba usage here to speed things up

Reply:
07c4b26d4

Reply:
Here's my favoured compromise: 2abaab447390948540b1edc567ed605d97be6fb4

Reply:
Ideal